draft-ietf-taps-interface.md

```diff
@@ -446,7 +446,7 @@ the lifetime of a connection using Transport Properties, as defined in
 Transport Properties are divided into Selection, Connection, and Message
 Properties. During pre-establishment, Selection Properties (see
 {{selection-props}}) are used to specify which paths and protocol stacks can be
-used and are preferred by the application, and Connection Properties (see
+used and are preferred by the application during candidate selection, and Connection Properties (see
 {{connection-props}}) can be used to influence decisions made during
 establishment and to fine-tune the eventually established connection. These
 Connection Properties can also be used later, to monitor and fine-tune
```

Comment:
Suggested change:
```
Selection Properties (see {{selection-props}}) can only be set during pre-establishment. They are only used to specify which paths and protocol stacks can be used and are preferred by the application. 
Connection Properties {{connection-props}}) can also be set and may be changed later and are used to influence decisions made during	establishment and to fine-tune the established connection. 
```

Reply:
@philsbln Your suggested text changes the "can be used to influence decisions" for Connection properties to "are used to influence decisions". I think we should not say "are used". Implementations may consider all Transport Properties, but do not have to use them. If it is something required by the application it should go in a Selection Property I think.

Reply:
@abrunstrom The application can use them to influence decisions – whether he implementation cares about anything that isn't "require" or "prohibit" or throws an error because the properties are unsupported is implementation specific. Influence does not mean that it is actually chnaging anything.

Reply:
I'd say /influence/ can change end results. Would prefer /inform/ to /influence/?

Reply:
I am also fine with inform.

Reply:
Yes, inform is better so let's use that either way. 
But I would still prefer "can be used" rather than "are used" also becuase there are two  parts to the sentence and the second part may also not happen. So I think "are used" does not read right. @philsbln was there a reason you changed it? I thought it was accidental when reformulating the text. You actually write that "applications can  use them to influence decisions" in your latest reply, so seems "can use" is also a natural formulation for you?

Reply:
@abrunstrom the reason I changed it was to avoid having six "can"s in four consecutive sentences.
I don't see that this "are used" suggests a necessary action except when you really want to do the thing these properties are designed for.

Reply:
For me, the original text didn't make a clear enough distinction when and how each set of properties is used. 
Thanks for the suggestion @philsbln, yours works even better.